app/controllers/vehicle_details_controller.rb

```diff
@@ -0,0 +1,33 @@
+class VehicleDetailsController < ApplicationController
+  before_action :init_new, only: %i[new create]
+  def new
+    @vehicle_detail = VehicleDetail.new
+  end
+
+  def create
+    @vehicle_detail = VehicleDetail.new(vehicle_detail_params)
+    if @vehicle_detail.save
+      flash[:success] = t "controller.vehicle.create.success"
+      redirect_to vehicle_details_path
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def models
+    brand_id = params[:brand_id]
+    @models = Model.by_brand(brand_id)
```

Comment:
```suggestion
    @models = Model.by_brand(params[:brand_id])
```
Sao không gộp thế này luôn